templates/common/baremetal/files/baremetal-keepalived.yaml

```diff
@@ -24,6 +24,32 @@ contents:
           path: "/etc/keepalived"
       - name: run-dir
         empty-dir: {}
+      - name: chroot-host
+        hostPath:
+          path: "/"
+      initContainers:
```

Comment:
Why do we need this init container?

I think that this change will break the unicast keepalived deployment.
In unicast mode the keepalived conf should be rendered in a synchronized manner (we use etcd for that purpose), otherwise, each master will render first a cfg file with empty **unicast_peers** section, so we'll end up with multiple nodes holding the VIP.

Reply:
This was added to template the check scripts. The monitor only regenerates the keepalived.conf file, but to make the check ports dynamic I had to template the scripts as well. I'll have to look at whether we can put the scripts in a separate location so we can template only them. Looks like I need to rebase this anyway so I'll look at it then.

Reply:
I tried the current version of this with unicast enabled and it worked correctly as far as I can tell. It's only templating the scripts in the init container, so it shouldn't affect the unicast configuration.